flake.nix

```diff
@@ -198,14 +207,36 @@
 
         release-files = import ./nix/release-files.nix { inherit self pkgs; };
 
+        # Common tests version - includes non-GC, non-release/debug specific tests.
+        common-systems-go = pkgs.releaseTools.aggregate {
+          name = "common-systems-go";
+          constituents =
+            pkgs.lib.attrValues common-constituents ++
+            pkgs.lib.attrValues checks ++
+            pkgs.lib.attrValues buildableReleaseMoPackages ++
+            filter_tests "common"  # Only include common tests
+            ++ builtins.attrValues js;
```

Comment:
Why do these aggregate jobs need to build all of those common packages? Wouldn't it be better to just have:
```nix

        # Common tests version - includes non-GC, non-release/debug specific tests.
        common-systems-go = pkgs.releaseTools.aggregate {
          name = "common-systems-go";
          constituents = filter_tests "common";  # Only include common tests
        };
```

Reply:
Ah, yes, that's correct, I just overlooked that. Will fix.